integration-tests/check-subscriptions

```diff
@@ -321,15 +307,16 @@ class TestSubscriptions(SubscriptionsCase):
         self.machine.execute("useradd junior; echo junior:foobar | chpasswd")
         self.login_and_go("/subscriptions", user="junior")
         self.browser.wait_in_text(".pf-c-empty-state__body", "current user isn't allowed to access system subscription")
-        self.allow_authorize_journal_messages()
+        self.allow_journal_messages("junior is not in the sudoers file.  This incident will be reported.")
+
```

Comment:
Extra empty line that will trigger E303. No, flake8 does not run on this file yet; regardless, I'd avoid adding more issues to fix later on.

Reply:
Fixes